### ISSUE TITLE
Ladders are blocks and cannot be placed down anywhere. I suggest changing them to be bricks.

### ISSUE THREAD

**Issue by Gliese-832-c** [tsunamayo/Starship-EVO#2999]
### Short Description of the issue:
For some reason ladders a blocks, not bricks. That means, that they can only be placed at certain positions. I suggest turning them into bricks instead, so you can offset them, and place them anywhere you want.

### Steps to reproduce the issue:
Try to place down a ladder.

### Environment
- Starship EVO version: early-access 20w36b
- Graphics Card: MSI RX 570 8G Armor OC

### Additional information:
*not needed, as this is a suggestion*

**Comment by Hecateus** [comment]
stretchable too.

**Comment by Xerma** [comment]
The thing here might be related to when we get crew as their pathfinding will be block based, hence the prefab doors we have (blast door/hatch)

**Comment by tsunamayo** [comment]
Yes it would make sense, but Xerma is right the pathfinding will be voxel based - so I might wait a bit for that. Thanks

**Comment by Gliese-832-c** [comment]
Maybe you could add two versions of the ladder? A block based one for the crew pathfinding, and a brick based one that's stretchable that is only for players but has better decorative uses?

**Comment by Gliese-832-c** [comment]
Thinking about it, I have a suggestion. What if we could do manual pathfinding instead? As in, a tool similar to the linker, where you can place down points in your hallways and rooms to make paths on which crew members will be able to walk? You should be able to toggle, either you let the game automatically pathfind, or you manually do it!

**Comment by WILDCATreactor** [comment]
You've got me thinking of using paint/stickers to create AI paths, but that only works for a certain subset of gritty mil-scifi
